akka-http-core/src/main/scala/akka/http/scaladsl/model/headers/headers.scala

```diff
@@ -629,6 +667,13 @@ final case class Referer(uri: Uri) extends jm.headers.Referer with ModeledHeader
   def getUri: akka.http.javadsl.model.Uri = uri.asJava
 }
 
+object `Timeout-Access` extends ModeledCompanion[`Timeout-Access`]
+final case class `Timeout-Access`(timeoutAccess: akka.http.scaladsl.TimeoutAccess)
```

Comment:
We could drop the namespace at the expense of another import.
So I opted for saving one line... :)